rnndb/graph/gm200_texture.xml

```diff
@@ -0,0 +1,390 @@
+		</bitfield>
+		<bitfield high="12" low="10" name="TILE_WIDTH_IN_GOBS" type="g80_gobs_per_block">
+			<doc>width of the sparse tile in gobs</doc>
+		</bitfield>
+		<bitfield pos="13" name="GOB_3D" type="boolean">
+			<doc>enable for 3D gobs</doc>
+		</bitfield>
+	</reg32>
+
+	<reg32 offset="0xc" name="3" varset="TIC_HEADER_VERSION" variants="PITCH_COLORKEY-">
+		<brief>these fields are available if (HEADER_VERSION != ONE_D_BUFFER)</brief>
+		<bitfield pos="16" name="LOD_ANISO_QUALITY_2" type="boolean">
+			<doc>enable more accurate aniso ratio to solve nv40 artifact. May reduce perf</doc>
+		</bitfield>
+		<bitfield pos="17" name="LOD_ANISO_QUALITY" type="g80_lod_quality">
+			<doc>enable use of 4 vectors for aniso lod calculation</doc>
+		</bitfield>
+		<bitfield pos="18" name="LOD_ISO_QUALITY" type="g80_lod_quality">
+			<doc>enable use of 4 vectors for iso lod calculation</doc>
+		</bitfield>
+		<bitfield high="20" low="19" name="ANISO_COARSE_SPREAD_MODIFIER" type="g80_aniso_spread_modifier">
+			<doc>spread modifier modifies coarse spread only, not sample count</doc>
+		</bitfield>
+		<bitfield high="25" low="21" name="ANISO_SPREAD_SCALE">
+			<doc>scale spread by 2^(x/32), adjust sample count accordingly. Applied after spread func</doc>
+		</bitfield>
+		<bitfield pos="26" name="USE_HEADER_OPT_CONTROL" type="boolean">
+			<doc>1: use optimization controls in header;  0: use optimization controls in sampler. The affected optimization controls are: MAX_ANISOTROPY; MIP_LOD_BIAS; TRILIN_OPT</doc>
+		</bitfield>
+		<bitfield pos="27" name="DEPTH_TEXTURE" type="boolean">
+			<doc>Tells texture unit to smear border color from R to RGBA channels</doc>
+		</bitfield>
+		<bitfield high="31" low="28" name="MAX_MIP_LEVEL">
+			<doc>max lod, typically = # mipmap levels - 1</doc>
+		</bitfield>
+	</reg32>
+
+	<reg32 offset="0x10" name="4_ONE_D_BUFFER" varset="TIC_HEADER_VERSION" variants="ONE_D_BUFFER">
+		<brief>these fields are available if (HEADER_VERSION == ONE_D_BUFFER)</brief>
+		<bitfield high="15" low="0" name="WIDTH_MINUS_ONE_BITS_15_TO_0">
+			<doc>least significant 16 bits of the "width minus one" field</doc>
+		</bitfield>
+	</reg32>
+
+	<reg32 offset="0x10" name="4" varset="TIC_HEADER_VERSION" variants="PITCH_COLORKEY-">
+		<brief>these fields are available if (HEADER_VERSION != ONE_D_BUFFER)</brief>
+		<bitfield high="15" low="0" name="WIDTH_MINUS_ONE">
+			<doc>least significant 16 bits of the "width minus one" field</doc>
+		</bitfield>
+		<bitfield high="21" low="19" name="ANISO_SPREAD_MAX_LOG2">
+			<doc>Log(aniso-filter spread)</doc>
+		</bitfield>
+		<bitfield high="31" low="29" name="BORDER_SIZE">
+			<value value="0" name="ONE"/>
+			<value value="1" name="TWO"/>
+			<value value="2" name="FOUR"/>
+			<value value="3" name="HEIGHT"/>
+			<value value="7" name="SAMPLER_COLOR">
+				<doc>border comes from sampler</doc>
+			</value>
+			<doc>border may be either constant color (from sampler), 1, 2, 4, or 8</doc>
```

Comment:
Cannot find more information about this, but your explanation is the only one that makes sense.